pseudonymsys/ca.go

```diff
@@ -15,16 +15,22 @@ import (
 type CA struct {
 	DLog            *dlog.ZpDLog
 	SchnorrVerifier *dlogproofs.SchnorrVerifier
-	caName          string
 	a               *big.Int
 	b               *big.Int
 	privateKey      *ecdsa.PrivateKey
 }
 
-func NewCA(caName string) *CA {
+type CACertificate struct {
```

Comment:
Because what we actually need is only CA endpoint (the same for credential issuer and verifier).